language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/builder/FieldCompletionItemBuilder.java

```diff
@@ -67,4 +67,21 @@ public static CompletionItem build(RecordFieldSymbol symbol) {
     public static CompletionItem build(ObjectFieldSymbol symbol) {
         return getCompletionItem(symbol);
     }
+
+    /**
+     * Build the constant {@link CompletionItem}.
+     *
+     * @param objectFieldSymbol {@link ObjectFieldSymbol}
+     * @param symbolName  Name of the symbol
+     * @return {@link CompletionItem} generated completion item
+     */
+    public static CompletionItem buildField(ObjectFieldSymbol objectFieldSymbol, String symbolName) {
+        String label = symbolName + "." + objectFieldSymbol.getName().get();
```

Comment:
Having  `isPresent` or `isEmpty` check  here in line 79 would be safe.

Reply:
Can't we guarantee that an object field always has a name, as per the grammar?

Reply:
I see. The `isPresent` check is not necessary